websites/F/Flowlab/presence.ts

```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({ clientId: "991160367629750372" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "icon",
+			startTimestamp: browsingTimestamp,
+		},
+		{ pathname, hostname } = document.location;
+
+	if (hostname === "flowlab.io") {
+		switch (pathname) {
+			case "/": {
+				presenceData.details = "Viewing home page";
+				break;
+			}
+			case "/game/list": {
+				presenceData.details = "Viewing their games";
+				break;
+			}
+			case "/game/browse": {
+				presenceData.details = "Viewing games page";
+				break;
+			}
+		}
+	}
+	if (pathname.includes("/profile")) {
+		presenceData.details = `Viewing ${document
+			.querySelectorAll(".username")
+			.item(0)
+			.textContent.trim()}'s profile`;
+	} else if (pathname.includes("/view/")) {
+		presenceData.details = `Editing "${
+			document.querySelector("#game_title").textContent
+		}"`;
+		presenceData.state = document.querySelector("#game_author").textContent;
+	} else if (pathname.includes("/play/")) {
+		(presenceData.details = `Playing "${
+			document.querySelector("#game_title").textContent
+		}"`),
+			(presenceData.state = `${
+				document.querySelector("#game_author").textContent
+			}`);
```

Comment:
```suggestion
		presenceData.details = `Playing "${
			document.querySelector("#game_title").textContent
		}"`;
			presenceData.state = `${
				document.querySelector("#game_author").textContent
			}`;
```

Reply:
I just committed this and the Compile and Lint Check failed:
```
yarn run v1.22.19
$ prettier --check . && eslint .
Checking formatting...
[warn] websites/F/Flowlab/presence.ts
[warn] Code style issues found in the above file. Forgot to run Prettier?
```